package.json

```diff
@@ -23,19 +23,20 @@
     "@types/node": "^13.1.0",
     "@typescript-eslint/parser": "^2.19.2",
     "eslint-plugin-typescript": "^0.14.0",
-    "fastify": "^2.11.0",
-    "iltorb": "2.4.3",
+    "fastify": "^3.0.0-alpha.1",
```

Comment:
What is the plan here, @mcollina? Are we going to release all of the core modules targeting the alpha releases and then come back and do _another_ pass to update them to the final v3?

Reply:
What would you prefer? I think it's the best path forward. Hopefully if dependabot works we can automate some of it.

Reply:
Sounds good. Let's learn how well dependabot helps out with this.